Community/Controllers/CommunitiesController.cs

```diff
@@ -26,14 +27,37 @@ public ActionResult CreateCommunity([FromBody]CreateCommunity createCommunity)
 
             return Created("", community);
         }
-        
+
+        // GET
         [HttpGet]
-        public ActionResult GetCommunities()
+        public ActionResult GetCommunities([FromQuery] string name, [FromQuery] string description, [FromQuery]DateTime data )
```

Comment:
Geralmente pra se referir a uma data (10/10/2010) usamos "date" e não "data"
```suggestion
        public ActionResult GetCommunities([FromQuery] string name, [FromQuery] string description, [FromQuery]DateTime date )
```

Reply:
Resolvido.